src/sweeper/backup.py

```diff
@@ -4,3 +4,19 @@
 backup.py
 A module that creates a gdb if it doesn't exist and inserts a feature class
 '''
+import arcpy 
+
+def backup_data(source_data, out_path):
+    # if the gdb does not exist, then create it
+    if not arcpy.Exists(out_path):
+        out_dir = out_path.rsplit('\\', 1)[0]
+        gdb_name = out_path.rsplit('\\', 1)[1]
```

Comment:
I think all of your rsplitting code has bugs

Reply:
it seems to be running okay.  does the syntax look incorrect?

Reply:
yup you're right